lib/ansible/plugins/terminal/dellos6.py

```diff
@@ -48,7 +48,7 @@ def on_authorize(self, passwd=None):
 
         cmd = {u'command': u'enable'}
         if passwd:
-            cmd[u'prompt'] = to_text(r"[\r\n]?password: $", errors='surrogate_or_strict')
+            cmd[u'prompt'] = to_text(r"[\r\n]?password:$", errors='surrogate_or_strict')
```

Comment:
Prexisting but note that you're still operating on a string literal here so you can write it like this:
``` python
cmd[u'promt'] = ur"[\r\n]?password:$"
```

Reply:
string literals are quicker than calling to_text() so that's why I mention it.